packages/orchestration/src/exos/cosmos-orchestration-account.js

```diff
@@ -749,9 +731,14 @@ export const prepareCosmosOrchestrationAccountKit = (
           return harden(coins.map(toDenomAmount));
         },
       },
+      /**
+       * TODO: transferWatcher can be removed once there is no instance of this
```

Comment:
This tag might help with garbage collection and IDE hints

```suggestion
      /**
       * @deprecated
       * TODO: transferWatcher can be removed once there is no instance of this
```

Reply:
Thanks!